### ISSUE TITLE
Accessibility issue: Button colors

### ISSUE THREAD

**Issue by PitWenkin** [Geoportail-Luxembourg/geoportailv3#3271]
The buttons on map.geoprtail.lu as well ass in embed maps only have a color contrast ration of 4.3:1 (#2980B9 on #fff respectivly inverted on hover). But should have a ration of 4.5:1 

A possible fix would be to add a second color to both
geoportal/jsapi/less/geoportail.api.less and geoportal/geoportailv3_geoportal/static-ngeo/less/geoportailv3.api.less
and use a media query to change the colors

```
@contrastcolor: #2980b9;

…

@media (prefers-contrast: more) {

    .lux-map .ol-control button,
    .lux-map .lux-bg-selector-select,
    .lux-map .lux-bg-selector::after{
        color: @contrastcolor;
    }

    .lux-map .ol-control button:hover {
        background-color: @contrastcolor;
    }
}
```

Note: This issue came up during an accessibility check for https://github.com/accessibility-luxembourg/ReferentielAccessibiliteMobile of an app using an embedded map.
